lib/scenario_runner.py

```
@@ -1236,21 +1240,29 @@ def _setup_services(self):
             # injection of unwawnted params
             docker_run_string = ['docker', 'run', '-it', '-d', '--name', container_name]
 
-            docker_run_string.append('-v')
 
             repo_mount_path = service.get('folder-destination', '/tmp/repo')
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/repo"

(S108)